extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeStreamExtractor.java

```diff
@@ -1035,4 +1035,29 @@ public String getThumbnailUrl() throws ParsingException {
             }
         };
     }
+
+    @Nullable
+    public StreamFrames getFrames() {
```

Comment:
Add a method in the base class `StreamExtractor` that returns `null` by default, then add `@Override`.